docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -73,6 +80,14 @@ code: |
 code: |
   MLH_time_min = 10
   MLH_time_max = 45
+---
+code: |
+  include_MLH_materials = False      # include a list of needed info to complete the tool?
```

Comment:
Recommend using "MLH_materials_included" to be consistent with other framework-level variables that all start with MLH.